lms/static/sass/elements/_dropdown.scss

```diff
@@ -0,0 +1,105 @@
+.wrapper-more-actions {
+    @include float(right);
+    @include margin-right(($baseline/2));
+    display: inline-block;
+    position: relative;
+
+    .button-more.has-dropdown {
+        @extend %ui-depth2;
+        position: relative;
+        height: ($baseline*2);
+        width: ($baseline*2);
+        padding: 0;
+        border: 1px solid transparent;
+        background: $white;
+        box-shadow: none;
+        font-size: $body-font-size;
+        text-shadow: none;
+        line-height: ($baseline*2);
+
+        .icon {
+            color: $gray-l1;
+        }
+
+        &:hover,
+        &:focus {
+            background: $white;
+            border-color: $gray-l3;
+
+            .icon {
+                color: $gray-d3;
+            }
+        }
+
+        &.is-active {
+            border-width: 1px 1px 0 1px;
```

Comment:
Can this be simplified to use ``border-bottom-width`` (instead of defining all of the values again)?